### PR TITLE
refactor: enable virtual scroll on the spectra list

### LINKED ISSUE #1981: Slow highlight in list of spectra panel
There seems to be a general problem with react-table component that makes it very slow. I wonder if this can be solved globally.

When we have a lot of spectra (1000 for example) and even if the spectra are not displayed it takes around 1s to refresh the list of spectra ?!

image

and it seems to be related to react-table

image

### PR DESCRIPTION
**Pull request by hamed-musallam**
close #1981

### CODE REVIEW

File: test-e2e/type/ClickOptions.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+interface ClickOptions {
```
Comment (targos) [inline test-e2e/type/ClickOptions.ts:1]: You should be able to use the types from playwright.

Comment (hamed-musallam) [reply test-e2e/type/ClickOptions.ts:1]: it seems that they did not expose it, this why I create an interface

Comment (targos) [reply test-e2e/type/ClickOptions.ts:1]: `type ClickOptions = Parameters<Page['click']>[1];`

Comment (hamed-musallam) [reply test-e2e/type/ClickOptions.ts]: Thanks  @targos

**Pull request merged**